src/harness/collectionsImpl.ts

```diff
@@ -50,6 +50,11 @@ namespace collections {
             return index >= 0 ? this._values[index] : undefined;
         }
 
+        public getKeyAndValue(key: K): [ K, V ] | [ undefined, undefined ] {

```

Comment:
I'd probably call this `getEntry` to align with the `entries()` method below, though I'd make the return value `[K, V] | undefined` since you end up allocating a useless two-element array for negative cases.

Reply:
Oops, missed the precedent.  Changed.